app/controllers/api/v1/invoices/item_invoice_controller.rb

```diff
@@ -0,0 +1,8 @@
+class Api::V1::Invoices::ItemInvoiceController < ApplicationController
```

Comment:
@tmikeschu Thats what I originally tried, but Rails was dumb and just loaded the default items_controller in the app/controllers folder rather than the items_controller in app/controllers/api/v1/invoices/items_controller.rb